node_modules/oae-version/tests/test-version.js

```diff
@@ -0,0 +1,77 @@
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var assert = require('assert');
+var fs = require('fs');
+var util = require('util');
+
+var RestAPI = require('oae-rest');
+var RestContext = require('oae-rest/lib/model').RestContext;
+var TenantsTestUtil = require('oae-tenants/lib/test/util');
+var TestsUtil = require('oae-tests');
+
+var UIAPI = require('oae-ui');
+var UIConstants = require('oae-ui/lib/constants').UIConstants;
+var UITestUtil = require('oae-ui/lib/test/util');
+
+describe('Version information', function() {
+
+    /**
+     * Test that verifies that the version information is returned
+     */
+    it('should return the version information', function(callback) {
+        // Create various rest contexts
+        var anonTenantRestContext = TestsUtil.createTenantRestContext(global.oaeTests.tenants.cam.host);
+        var adminTenantRestContext = TestsUtil.createTenantAdminRestContext(global.oaeTests.tenants.cam.host);
+        var anonGlobalRestContext = TestsUtil.createGlobalRestContext();
+        var globalAdminRestContext = TestsUtil.createGlobalAdminRestContext();
+        TestsUtil.generateTestUsers(adminTenantRestContext, 1, function(err, users, user) {
+            assert.ok(!err);
+            var userTenantRestContext = user.restContext;
+
+            // Verify the version information on regular tenancies
+            verifyVersionInformation(anonTenantRestContext, function() {
+                verifyVersionInformation(userTenantRestContext, function() {
+                    verifyVersionInformation(adminTenantRestContext, function() {
+
+                        // Verify the version information on the global admin
+                        verifyVersionInformation(anonGlobalRestContext, function() {
+                            verifyVersionInformation(globalAdminRestContext, callback);
+                        });
+                    });
+                });
+            });
+        });
+    });
+
+    /*!
+     * Verify the version information
+     *
+     * @param  {RestContext}        restContext     The rest context to get the version information with
+     * @param  {Function}           callback        Standard callback function
+     * @throws {AssertionError}                     Thrown if any assertions fail
+     */
+    function verifyVersionInformation(restContext, callback) {
+        RestAPI.Version.getVersion(restContext, function(err, version) {
+            assert.ok(!err);
+            assert.ok(_.isObject(version));
```

Comment:
Is this really supposed to be an object? Didn't we output a string from the REST endpoint? My guess is we're getting bitten by `_.isObject` working on strings :/